src/main/java/com/artipie/npm/proxy/RxNpmProxyStorage.java

```diff
@@ -139,7 +141,11 @@ public Completable save(final NpmAsset asset) {
                     new NpmPackage(
                         name,
                         new String(content.array(), StandardCharsets.UTF_8),
-                        metadata.getString("last-modified")
+                        metadata.getString("last-modified"),
```

Comment:
@brastak I believe `json` parsing of `metadata` should be done inside `NpmPackage`. This will make 
`NpmPackage` more than just a bag of data and its internal metada structure will remain inside that object and not spread through the codebase.